minutes/_project/2020/2020-11-24.md

```diff
@@ -8,8 +8,8 @@ tags: project management, CSSI, minutes
 
 ### Video Conference Information
 * Instant messaging: [Matrix](https://element.im/app/#/room/#plasmapy:openastronomy.org) and [Gitter](https://gitter.im/PlasmaPy/Lobby)
-* [GitHub Minutes Repository](https://github.com/PlasmaPy/plasmapy-project)
-    * [CSSI Sub-directory](https://github.com/PlasmaPy/plasmapy-project/tree/master/CSSI_2020-present)
+* [GitHub Minutes Repository](https://github.com/PlasmaPy/plasmapy-project/tree/master/minutes)
+    * [CSSI Sub-directory](https://github.com/PlasmaPy/plasmapy-project/tree/master/minutes/_project)
```

Comment:
A thought would be to change the "CSSI meeting" name to be maybe like an "education and community outreach meeting" that could involve more people.  In any case, that's for the future, and not this PR.

Reply:
Yeah, since nothing is completely hashed out yet, I thought `_project` was a nice grab bag for the meeting.  Also, the CSSI meeting is currently an overarching project meeting.